cpp/src/glm/qn/simple_mat/sparse.hpp

```diff
@@ -25,10 +25,13 @@
 #include <raft/handle.hpp>
 #include <raft/linalg/binary_op.cuh>
 #include <raft/linalg/cublas_wrappers.h>
+
+// TODO: Need to wrap the cusparse functions in the RAFT public API
```

Comment:
Is there a RAFT issue to track it?

Reply:
Thee is: https://github.com/rapidsai/raft/issues/475